tests/app/notifications/test_process_notification.py

```diff
@@ -276,10 +275,6 @@ def test_send_notification_to_queue(
     if '.' not in expected_task:
         expected_task = f'provider_tasks.{expected_task}'
     mocked = mocker.patch(f'app.celery.{expected_task}.apply_async')
-    Notification = namedtuple(
-        'Notification',
-        ['id', 'key_type', 'notification_type', 'reply_to_text', 'created_at']
-    )
```

Comment:
Previously was using a tuple to avoid using the regular `Notification` model but the model is actually needed now that we have a new method `sends_with_custom_number`